scripts/actions/check-job-progress.js

```diff
@@ -0,0 +1,35 @@
+const fetchAndDeserialize = require('./fetch-and-deserialize');
+const { vendorRequest } = require('./utils/vendor-request');
+
+const checkJobStatus = async (batchUid) => {
+  const {
+    response: {
+      data: { status },
+    },
+  } = await vendorRequest(
+    'GET',
+    `https://api.smartling.com/jobs-api/v3/projects/${project}/jobs/${batchUid}`
+  );
+
+  return status === 'COMPLETED';
+};
+
+const main = async () => {
+  //get list of current job batch uids from dyanmoDB, for each one execute checkJobStatus
+  try {
+    const isCompleted = checkJobStatus(batchUid);
+
+    if (isCompleted) {
+      await fetchAndDeserialize(batchUid);
+      process.exit(0);
+    } else {
+      process.exit(1);
```

Comment:
If these are not completed, I want to stop the job and I'm not sure what the best way to do this is.

Reply:
I see `process.exit(1);` as a way to say "something went wrong with this script. Perhaps we simply want to log out that the job is still being processed and exit with a `0` code (success).

I also worry that failed actions might blow up our email.